src/main/java/com/studypals/domain/studyManage/dto/StudyStatusRes.java

```diff
@@ -0,0 +1,6 @@
+package com.studypals.domain.studyManage.dto;
+
+/**
+ * 공부 상태 반환
+ */
+public interface StudyStatusRes {}
```

Comment:
[P3]
공부 중인 경우와, 공부 중이지 않은 경우의 응답 dto 를 분리할 필요가 있을까 싶습니다. studying 만 true/false 로 두고 나머지는 null 로 반환, 혹은 null 인 경우 json 직렬화 옵션을 끄는 방안은 어떠신지 여쭙고 싶습니다.

```suggestion
@JsonInclude(JsonInclude.Include.NON_NULL)
public record StudyStatusRes (
    Boolean studying,
    LocalDateTime startTime,
    Long categoryId,
    String name,
    Long goal
) {}
```

Reply:
다시 생각해보니까 dto 주제에 전략 패턴까지 들어갈 필요는 없을 거 같네요, 그냥 단일 dto로 유지하고 필드를 null로 두는 방향으로 수정하겠습니다.